config/applications.json

```diff
@@ -111,6 +111,14 @@
 		"link": "https://learn.microsoft.com/en-us/sysinternals/downloads/autoruns",
 		"winget": "Microsoft.Sysinternals.Autoruns"
 	},
+	"WPFInstallrdcman": {
+		"category": "Microsoft Tools",
+		"choco": "rdcman",
+		"content": "Remote Desktop Connection Manager (RDCMan)",
+		"description": "RDCMan manages multiple remote desktop connections. It is useful for managing server labs where you need regular access to each machine such as automated checkin systems and data centers.",
+		"link": "https://learn.microsoft.com/en-us/sysinternals/downloads/rdcman",
+		"winget": "Microsoft.Sysinternals.RDCMan"
+	},
```

Comment:
RDCMan... that is a package I haven't seen in quite some time. I used to use this on the daily when I had 10+ Windows servers I was remoting in all the time to.

Reply:
It's quite a cool program, haven't used it but I can see it usefulness in Colleges and Data Centers for example.